include/triton/Analysis/Utility.h

```diff
@@ -189,18 +189,16 @@ bool supportMMA(triton::DotOp op, int version);
 
 bool supportMMA(Value value, int version);
 
+bool cvtReordersRegisters(RankedTensorType srcTy, RankedTensorType dstTy);
```

Comment:
Add a brief comment to this?

Reply:
(In particular does the MmaToDot shortcut also just reorder registers?  If so maybe the name for this should have "LinearLayout" or "LL" in the name?)

Reply:
What do you mean "LL" in the name?

Reply:
> In particular does the MmaToDot shortcut also just reorder registers?

There are some hacks. If we get true `matchMmaV3AndDotOperandLayout`, actually we already use warp shuffle, but this is a very special case

Reply:
"LL" meant LinearLayout.  Anyway this is good, thanks.  :)